src/Parser/GeoJsonParser.js

```diff
@@ -31,17 +31,12 @@ const firstPtIsOut = (extent, aCoords, crs) => {
     return !extent.isPointInside(coord);
 };
 const toFeature = {
-    populateGeometry(crsIn, coordinates, geometry, collection, feature) {
+    populateGeometry(crsIn, coordinates, geometry, feature) {
         geometry.startSubGeometry(coordinates.length, feature);
-
-        const useAlti = !collection.overrideAltitudeInToZero &&
-                        collection.size == 3 &&
-                        typeof coordinates[0][2] == 'number';
-
+        coord.crs = crsIn;
         // coordinates is a list of pair [[x1, y1], [x2, y2], ..., [xn, yn]]
         for (const pair of coordinates) {
```

Comment:
Maybe the name `pair` is no longer adapted, since in a 3d context we have 3 coordinates

Reply:
yes